build/sbt

```diff
@@ -20,10 +20,12 @@
 # When creating new tests for Spark SQL Hive, the HADOOP_CLASSPATH must contain the hive jars so
 # that we can run Hive to generate the golden answer.  This is not required for normal development
 # or testing.
-for i in "$HIVE_HOME"/lib/*
-do HADOOP_CLASSPATH="$HADOOP_CLASSPATH:$i"
-done
-export HADOOP_CLASSPATH
+if [ "$HIVE_HOME" ]; then
```

Comment:
Not a bash expert, but should this be using something like `if [ -d "$HIVE_HOME" ];` to check that it's actually a directory and exists, similar to how we check for the existence of the assembly directory in https://github.com/apache/spark/blob/5952bdb7df20d007d59f82261095faca3822c6f6/bin/spark-class#L51?